colors/build.js

```diff
@@ -2,29 +2,43 @@ const fs = require('fs');
 const colors = require('./photon.json');
 const metadata = require('./package.json');
 
-function createColor(color, element) {
-  var rv = [];
-  for (var variant in element) {
+function createColor(color, element, prefix) {
+  const rv = [];
+
+  for (const variant in element) {
     if (element.hasOwnProperty(variant)) {
-      var value = element[variant];
-      rv.push(`  --${color}-${variant}: ${value};\n`);
+      const value = element[variant];
+      rv.push(`${prefix}${color}-${variant}: ${value};\n`);
     }
   }
+  rv.push('\n');
   return rv;
 }
 
-let rv = [`/* Firefox Colors CSS Variables v${metadata.version} */
+
+const cssOutput = [`/* Firefox Colors CSS Variables v${metadata.version} */
 
 :root {
 `];
-for (var color in colors) {
+
+const sassOutput = [`/* Firefox Colors SCSS Variables v${metadata.version} */
+
+`];
+
+for (const color in colors) {
   if (colors.hasOwnProperty(color)) {
-    var element = colors[color];
-    rv.push(...createColor(color, element));
+    const element = colors[color];
+    cssOutput.push(...createColor(color, element, '  --'));
```

Comment:
Awww, no default parameter?  😉  (This is another one of those where I don't really mind if we don't change it.)